examples/next/pages/ui/components/amplify-authenticator/sign-in-with-email/index.page.tsx

```diff
@@ -0,0 +1,15 @@
+import { AmplifyAuthenticator, translations } from '@aws-amplify/ui-react';
+import awsExports from '@environments/auth-with-email/src/aws-exports';
+import { Amplify, I18n } from 'aws-amplify';
+
+Amplify.configure(awsExports);
+
+I18n.putVocabularies(translations);
```

Comment:
This should be removed, since the legacy translations aren't compatible with the new ones:

```suggestion
```